src/rest.cpp

```diff
@@ -307,32 +308,33 @@ static bool rest_block(const std::any& context,
         if (chainman.m_blockman.IsBlockPruned(*pblockindex)) {
             return RESTERR(req, HTTP_NOT_FOUND, hashStr + " not available (pruned data)");
         }
+        pos = pblockindex->GetBlockPos();
     }
 
-    if (!chainman.m_blockman.ReadBlockFromDisk(block, *pblockindex)) {
+    std::vector<uint8_t> block_data{};
+    if (!chainman.m_blockman.ReadRawBlockFromDisk(block_data, pos)) {
```

Comment:
In 3d7ae60f21:

Same discussion as with the previous commit. `ReadBlockFromDisk()` checks PoW and the signet signature after reading the block from disk while `ReadRawBlockFromDisk()` doesn't.

I agree on not performing these checks but would be nice to describe the changes in the commit description.

Reply:
Done, added a commit description addressing this.